style.css

```diff
@@ -0,0 +1,12 @@
+
+body{
+    background-color: darkgray;
+}
+h2{
+    color: blueviolet;
+    
+    font-family: Arial, Helvetica, sans-serif;
+    align-items: center;
+    padding-top: 40px;
+    margin-left: 200px;
+}   
```

Comment:
- [ ] Please do well your linters configuration so that it can track errors in your files you can refer to this [Link](https://github.com/microverseinc/linters-config/tree/master/html-css)

Reply:
I am confused about how to add linters. I asked about it and still confused

Reply:
Yes I followed the instruction and now linters is configured

Reply:
I also want to ask how can I change my PR title. When I follow the link, according to the link I should add comments in the file. Am I right??